go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-retryablehttp v0.6.7
 	github.com/hashicorp/hcl v1.0.0
+	github.com/hashicorp/terraform-plugin-sdk v1.15.0 // indirect
```

Comment:
please remove this dependency, because we're using `github.com/hashicorp/terraform-plugin-sdk/v2`.

Reply:
removed in next commit